nucypher/policy/conditions/lingo.py

```diff
@@ -330,6 +352,18 @@ def __init__(self, operand: Condition):
     "float": lambda a: float(a),
     "int": lambda a: int(a),
     "str": lambda a: str(a),
+    # JSON conversion
+    "fromJson": lambda a: json.loads(a),
+    "toJson": lambda a: json.dumps(a),
+    # hex conversion
+    "fromHex": lambda a: bytes(HexBytes(a)),
+    "toHex": _to_hex,
+    # hashing
+    "keccak": lambda a: keccak(
+        a.encode()
+        if isinstance(a, str)
+        else (a if isinstance(a, (bytes, bytearray)) else str(a).encode())
+    ),
```

Comment:
Let's use `keccak()` as is without casting to string since it already handles the relevant types for you. The only reason we need to convert a string to bytes is because if you are passing string to `keccak` you are supposed to use the `text=` kwarg; instead we convert it to bytes which `keccak` would have done internally anyway.

```suggestion
    "keccak": lambda a: keccak(a.encode() if isinstance(a, str) else a)
```